cibyl/config.py

```diff
@@ -0,0 +1,69 @@
+"""
+#    Copyright 2022 Red Hat
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+"""
+import logging
+import os
+import sys
+from collections import UserDict
+
+import yaml
+
+from cibyl.files import get_first_available_file
+
+LOG = logging.getLogger(__name__)
+
+
+class Config(UserDict):
+    """
```

Comment:
Would be nice to add some description here, even if it's just "Represents configuration handler..."